lib/jasmine-2.3.4/boot.js

```diff
@@ -0,0 +1,121 @@
+/**
```

Comment:
don't commit these files - place any files that you didn't write yourself in a separate directory, and place that in gitignore.